### PR TITLE
chore: refactor and re-theming document component

### PR DESCRIPTION
**Pull request by mwdchang**
### Summary
This does a round of refactoring on the Document component, reorganizes the visual layout to be more aligned with the intended design specs.

Note the operations are a little awkward at the moment:
- This needs the collapsible panel which is not yet in place
- The design does not match to the data format that we are working with

Both of these points above will be dealt with in subsequent PRs.

image

### Testing
This is available in both
- Data explorer, select a document search result item
- In the project overview, select from the paper menu in the sidebar

### CODE REVIEW

**Comment by YohannParis** [comment]
Please assign someone to review it, someone from the task is preferable like Tom.

**Review by amostafa-uncharted — COMMENTED** [review]
LGTM

File: packages/client/hmi-client/src/views/DataExplorer.vue [hunk 1]
```diff
@@ -91,19 +91,13 @@
...
-				:project="resources.activeProject"
-			>
-				<template #footer>
-					Add to cart
-					<br />
-					Add to Project
-				</template>
-			</document>
+			<div v-if="previewItem" class="selected-resources-pane">
+				<document :asset-id="previewItemId" :project="resources.activeProject" />
```
Comment (YohannParis) [inline packages/client/hmi-client/src/views/DataExplorer.vue:97]: `<Document>` to make it easier to see that this is a component.

File: packages/client/hmi-client/src/views/Document.vue [hunk 2]
```diff
@@ -1,91 +1,92 @@
 <template>
 	<section class="doc-view-container">
 		<div v-if="doc">
+			<div class="journal">{{ doc.journal }}</div>
 			<div v-if="docLink" class="title">
 				<a :href="docLink" target="_blank" rel="noreferrer noopener">{{ doc.title }}</a>
```
Comment (YohannParis) [inline packages/client/hmi-client/src/views/Document.vue:6]: We should remove `target="_blank"`. It's up to the user to decide how to open a link.

**Review by YohannParis — APPROVED** [review]
LGTM

**Review by YohannParis — DISMISSED** [review]

**Review by YohannParis — APPROVED** [review]

**Pull request merged**